modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -28,11 +41,60 @@ export interface UploadResult {
 }
 
 interface EditorUpload {
+  /**
+   * Cache of blob elements created in an editor instance.
+   *
+   * @property blobCache
+   * @type Object
+   */
   blobCache: BlobCache;
+
+  /**
+   * Adds a custom filter that controls the images which are included in the scan.
+   * Images must return true on every added filter to be considered valid.
```

Comment:
```suggestion
   * Adds a custom filter that will be called all images being uploaded.
   * Return `true` for an image that is considered to be valid
```
Not sure how this would sound compared to the original.

Reply:
Filters being included in the scan rather than upon uploading is more accurate, as the scan is performed to find all images when upload is called.